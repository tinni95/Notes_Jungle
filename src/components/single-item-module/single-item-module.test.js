import { shallow } from "enzyme";
import React from "react";
import ItemModule from "./single-item-module.component";

describe("with valid title", () => {
  const title = "title";
  const Item = shallow(<ItemModule title={title} />);
  it("expect to render ", () => {
    expect(Item.isEmptyRender()).toBe(false);
    expect(Item).toMatchSnapshot();
  });
});

describe("with invalid title", () => {
  const title = "";
  const Item = shallow(<ItemModule title={title} />);
  it("expect to render ", () => {
    expect(Item.isEmptyRender()).toBe(true);
    expect(Item).toMatchSnapshot();
  });
});
