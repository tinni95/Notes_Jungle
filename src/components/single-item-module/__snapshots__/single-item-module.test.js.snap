// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with invalid title expect to render  1`] = `ShallowWrapper {}`;

exports[`with valid title expect to render  1`] = `ShallowWrapper {}`;
